sys-apps/ignition/files/0005-config-v3_4-convert-ignition-2.x-to-3.1.patch

```diff
@@ -1,81 +0,0 @@
-From 5c730b1c2928f8246237b3d378d6bd6815a12e2c Mon Sep 17 00:00:00 2001
-From: Mathieu Tortuyaux <mathieu@kinvolk.io>
-Date: Tue, 27 Jul 2021 11:46:49 +0200
-Subject: [PATCH 05/16] config/v3_4: convert ignition 2.x to 3.1
-
-if the version of ignition is 2.x we convert it to 3.1 using
-ign-converter.
-
-it shoud support any 2.x version (or at the least the last 2 releases)
-
-Signed-off-by: Mathieu Tortuyaux <mathieu@kinvolk.io>
----
- config/v3_4_experimental/config.go | 38 ++++++++++++++++++++++++++++++
- 1 file changed, 38 insertions(+)
-
-diff --git a/config/v3_4_experimental/config.go b/config/v3_4_experimental/config.go
-index 512fc9f0..b0375236 100644
---- a/config/v3_4_experimental/config.go
-+++ b/config/v3_4_experimental/config.go
-@@ -15,6 +15,10 @@
- package v3_4_experimental
- 
- import (
-+	"encoding/json"
-+	"fmt"
-+
-+	"github.com/flatcar-linux/ignition/config/v2_4"
- 	"github.com/flatcar-linux/ignition/v2/config/merge"
- 	"github.com/flatcar-linux/ignition/v2/config/shared/errors"
- 	"github.com/flatcar-linux/ignition/v2/config/util"
-@@ -25,6 +29,8 @@ import (
- 
- 	"github.com/coreos/go-semver/semver"
- 	"github.com/coreos/vcontext/report"
-+
-+	"github.com/flatcar-linux/ign-converter/translate/v24tov31"
- )
- 
- func Merge(parent, child types.Config) types.Config {
-@@ -67,6 +73,38 @@ func ParseCompatibleVersion(raw []byte) (types.Config, report.Report, error) {
- 		return types.Config{}, rpt, err
- 	}
- 
-+	// if the version is 2.x or 1.x, we
-+	// convert it to 3.1
-+	if version.Major != 3 {
```

Comment:
Same here

Reply:
Same as above, the actual used patch is: `sys-apps/ignition/files/0005-config-v3_4-convert-ignition-2.x-to-3.4-exp.patch` - name are differing but same content.
Commit has been amended back in the days.